### PR TITLE
Simplify bias_add in Conv1D

### PR DESCRIPTION
**Pull request by lgeiger**
`tf.nn.bias_add` has full support for `NCHW` and `NHWC` so we don't need this special case any more.

### CODE REVIEW

**Comment by lgeiger** [comment]
@tanzhenyu Would be cool to get a review for this.

**Review by tanzhenyu — COMMENTED** [review]
It'd be great to have a unit test for this -- it seems the original if has a comment on why it needs to reshape bias?

**Comment by gbaned** [comment]
@lgeiger Could you please check reviewer comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
